baseline-cli/build.gradle.kts

```diff
@@ -3,9 +3,14 @@ plugins {
     alias(libs.plugins.shadow)
 }
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
     implementation(projects.sarif)
     implementation(libs.gson)
+    implementation("com.github.ajalt.clikt:clikt:4.3.0")
```

Comment:
For uniformity, please use the same approach as the other libs, via the `gradle/libs.versions.toml` file